webrtc-stats.html

```diff
@@ -2711,6 +2713,26 @@ <h2>
                   but is a single concealment event.
                 </p>
               </dd>
+              <dt>
+                <dfn><code>deceleratedSamples</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  The total number of samples inserted in order to decelerate playout speed. This
+                  can happen to prevent the jitter buffer from becoming empty.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>acceleratedSamples</code></dfn> of type <span class=
+                "idlMemberType"><a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  The total number of samples discarded in order to accelerate playout speed. This
+                  can happen to prevent the jitter buffer from becoming full.
```

Comment:
When playout is speeded up, this counter is increased by the difference between the number of samples received and the number of samples played out. If speedup is achieved by discarding samples, this will be the count of samples discarded."

Reply:
Name: removedSamples
Rephrase: Accelerate is a consequence, not a definition.

Reply:
Relating to my comment above, saying "removed" is better than saying "discarded", because discarded sounds like that data was not examined at all.

Reply:
Suggestion:

Name: removedDeceleratedSamples
Description: Harald's definition but replace "discarded" with "removed"